packager/signer/signer_test.go

```diff
@@ -78,7 +78,7 @@ type SignerSuite struct {
 
 func (this *SignerSuite) new(urlSets []util.URLSet) http.Handler {
 	forwardedRequestHeaders := []string{"Host", "X-Foo"}
-	handler, err := New(fakeCertHandler{}, pkgt.Key, urlSets, &rtv.RTVCache{}, func() error { return this.shouldPackage }, nil, true, forwardedRequestHeaders)
+	handler, err := New(fakeCertHandler{}, pkgt.Key, "", urlSets, &rtv.RTVCache{}, func() error { return this.shouldPackage }, nil, true, forwardedRequestHeaders)
 	this.Require().NoError(err)
```

Comment:
Optionally, a test for this functionality would be cool (make a new subdir under testdata with an html file, I guess). But no big deal if it turns out to be too ornery to set up.